### DISCUSSION TITLE
WSL install says the requested operation requires elevation, installs anyway

### DISCUSSION THREAD

**Discussion by givihuda** [microsoft/WSL#11896] [Q&A]
I'm on a Windows 11 computer (latest updates all around) and trying WSL on it. I tried Windows Terminal (WT) from the Microsoft Store, which opened Windows PowerShell (WPS) (so WT is just a wrapper for WPS?), then I typed `wsl`. That gave me some nice how-to output, and based on that I ran this:

`PS C:\Users\Me> wsl --install
The requested operation requires elevation.
Installing: Virtual Machine Platform
Virtual Machine Platform has been installed.
Installing: Windows Subsystem for Linux
Windows Subsystem for Linux has been installed.
Installing: Ubuntu
Ubuntu has been installed.
The requested operation is successful. Changes will not be effective until the system is rebooted.
PS C:\Users\Me>`

So... the requested operation requires elevation, then the requested operation is successful. I'm confused 😅 I'm glad it was successful, but this looks contradictory, and I'm concerned now that I'll have a bunch of snags to troubleshoot because it may be missing things it "required". 

Or is the error just wrong? What exactly in the installation needs elevation?

If I run it again like `sudo wsl --install`, a suggestion I found here - https://github.com/microsoft/WSL/issues/9032#issuecomment-1817586564, though it's a year old, and I have no idea if that will even be recognized in my WPS at the moment. Will that just do a full reinstall, or what? Honestly I'm just trying to get Node.js + nvm installed for VS Code, and I'd much rather use a *nix-like for it.

**Comment by OneBlue** [comment]
@givihuda: This message means that an elevation prompt will be shown. It's typically there to tell the user that the elevation is required (useful in cases where the elevation prompt is in background for instance). 

This is the expected behavior when wsl --install is invoked from a non-elevated context
